setup.py

```diff
@@ -44,7 +44,7 @@ def run(self):
 
 description = 'Convex Optimization Modeling for Microstructure Informed Tractography (COMMIT)'
 opts = dict(name='dmri-commit',
-            version='1.3.9.2',
+            version='1.4.0.0',
```

Comment:
The numbering format in `CHANGELOG.md` is different

Reply:
Done!